source/concepts/protocols.rst

```diff
@@ -0,0 +1,284 @@
+upon. The balances locked in the parent layer will be redistributed to the
+participants as per the balances in the settled state.
+
+This phase consists of three sub-phases:
+
+1. Finalize
+2. Register dispute on-chain
+3. Progress channel on-chain
+
+1. Finalize
+```````````
+
+In this sub-phase, the protocol tries to create an agreement on the state to be
+settled without any on-chain transactions. If it succeeds, then the channel
+direclty progresses to the settle phase. If not, then it transitions to the
+next sub-phase. The protocol is same for all three types of channels.
+
+If the latest off-chain state was marked as ``final`` already in the transact
+phase, it implies both the participants have agreed it to be state that needs
+to be settled. If it was not, another another off-chain transaction is sent, on
+behalf of the participant who initiated the channel settlement, marking the
+latest off-chain state as ``final``. It is up to the other participant to
+accept or reject it.
+
+If accepted, channel transitions to ``Settle phase``, if not it transitions to
+the next sub-phase.
+
+.. image:: ../_generated/concepts/register_generic_finalize.svg
+  :align: Center
+  :alt: Image not available
+
+
+2. Register dispute on-chain
+````````````````````````````
+
+In this sub-phase, the protocol creates an agreement on the state  to be
+settled by registering a dispute with the parent layer and resolving it.
+
+The parent layer for registering dispute is always the blockchain. In case of
+
+1. Ledger channel: The state of the channel, all its sub-channels and virtual
+   channels must be collected and registered on the blockchain.
+
+2. Sub-channel: The state of the parent channel, all the sub-channels and
+   virtual channels of the parent channel must be collected and registered on
+   the blockchain.
+
+3. Virtual channel: The state of the parent channel between the participant who
+   initiated channel settlement and the intermediary, all the sub-channels and
+   virtual channels of this parent channel must be collected and registered on
+   the blockchain.
+
+   Once intermediary is notified that one of the parent channels holding funds
+   for the virtual channel is registered on-chain, the intermediary will try to
+   finalize the state of the sub-channel between the intermediary and the other
+   channel participant (who did not initiate settlement) through an off-chain
+   transaction. If is succeeds, then that sub-channel will be closed. If not,
+   then the ledger channel backing that sub-channel, along with all its
+   sub-channels will be registered on the blockchain.
+
+
+.. image:: ../_generated/concepts/register_generic_register.svg
+  :align: Center
+  :alt: Image not available
+
+After the challenge duration for register expires, if the channel has
+
+    1. ``No app``, then this state can be settled.
+    2. ``An app``, the channel transitions to force-execution sub-phase.
```

Comment:
consistent markup for  phases and sub-phases / steps throughout this doc would ease reading. E.g., above there was 
> it transitions to ``Settle phase``.
in contrast to "force-execution" here, which I assume to be also the name of a particular phase,

Reply:
I have used ``settle phase`` style mark up consistently in all places.
Changes added in 7aea479.